action/entrypoint.sh

```diff
@@ -2,6 +2,17 @@
 
 # Enable the matcher.
 ACTION_FOLDER=$(dirname ${0})
+echo " - error_classes: $INPUT_ERROR_CLASSES"
```

Comment:
really nitpicking here, sorry: should be `${INPUT_ERROR_CLASSES}` :D (similar for the other cases closeby)

Reply:
done